lib/chatroom.js

```diff
@@ -0,0 +1,32 @@
+
+
+
+class Chatroom {
+    constructor() {
+        this.chatroom =  new Map();
+        this.userNumber = 1;
+    }
+
+    add(obj) {
+        obj.username = `USER- ${this.userNumber++}`;
```

Comment:
```suggestion
        obj.username = `USER${this.userNumber++}`;
```
No spaces makes this easier to type out and avoid errors when parsing the message.